tests/conftest.py

```diff
@@ -115,7 +115,12 @@ def RE():
     RE.subscribe(
         VerbosePlanExecutionLoggingCallback()
     )  # log all events at INFO for easier debugging
-    return RE
+    yield RE
+    try:
+        RE.halt()
```

Comment:
It might be better to use [RE.abort()](https://blueskyproject.io/bluesky/generated/bluesky.run_engine.RunEngine.abort.html#bluesky.run_engine.RunEngine.abort) here as I think this would still do cleanup

Reply:
I explicitly don't want to do cleanup :)